src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -457,6 +457,14 @@ const parseObjectKeyValueToMongoObjectKeyValue = (restKey, restValue, schema) =>
     );
   }
   value = mapValues(restValue, transformInteriorValue);
+
+  if (
+    typeof value === 'object' &&
+    '_bsontype' in value &&
+    !value[Symbol.for('@@mdb.bson.version')]
+  ) {
+    value[Symbol.for('@@mdb.bson.version')] = 5.0;
+  }
```

Comment:
Hi @mtrezza , i work with @coratgerl , we will take a look about this seems weird to me aslo.

Reply:
Please feel free; I would be fine with merging it into Parse Server 7, i.e. we can merge it on Nov 1st, see https://github.com/parse-community/parse-server/pull/8761#discussion_r1375255686. But any help on this matter is much appreciated.

Reply:
@Moumouls @coratgerl we'll go ahead and merge this for Parse Server 7

Reply:
We need definitely investigate failures without this code @mtrezza before merging, until we have a clear explanation and in case of this code need to be kept a detailed comment about why we have this.

I'll try to take a look this week with @coratgerl , we will do our best !

Reply:
I agree, that would he great.